naff/models/discord/message.py

```diff
@@ -559,27 +560,29 @@ async def edit(
         if message_data:
             return self._client.cache.place_message_data(message_data)
 
-    async def delete(self, delay: Absent[Optional[int]] = MISSING) -> None:
+    async def delete(self, delay: int = 0, *, context: "InteractionContext") -> None:
```

Comment:
This isn't as optional as you'd like it to be, and tests are failing because of it

Reply:
whoops. excuse me while i shame myself for that one

Reply:
Resolved in bc3e79220e57a97173368691b4b764a170a8a2e0